### DISCUSSION TITLE
NodeRed 3

### DISCUSSION THREAD

**Discussion by euDominic** [rdmtc/RedMatic#554] [General]
Hallo,

wird es hier noch ein Update geben? oder ist das Projekt tot?

eine Antwort wäre sehr freundlich.

**Comment by filmfacts** [comment]
Hallo,

ich schließe mich der Frage an. Nachdem heute ein Update von node-red-contrib-virtual-smart-home dazu geführt hat, dass meine komplette Smarthome-Steuerung per Sprache nicht mehr funktioniert, da der Node zwingend NodeRed 3 benötigt.

Ist hier noch ein Update geplant, oder sollte ich mich um eine andere Lösung bemühen? Und ganz ehrlich: ich würde sogar einen kleinen Obolus spenden.

Viele Grüße
Andreas

**Reply by euDominic** [reply]
das Projekt dürfte tot sein und es kommt auch keine Rückmeldung mehr. 

ich betreibe mittlerweile alles per proxmox.

ccu, home assistant, node-red und anderes alles virtualisiert auf einem Gerät und bin somit unhabeniger von toten Projekten

**Reply by filmfacts** [reply]
Ja, habe auch nirgendwo etwas gefunden. Nun hat der Raspberry Pi neben dem PiHole noch ein Node Red bekommen und ich konnte nach ein paar Stunden Bastelei wieder meine Smarthome-Aktoren per Alexa steuern. Schade, fand es ganz bequem, das Ganze auf der Homematic CCU zu haben.

**Comment by hobbyquaker** [comment]
Das Projekt lebt: RedMatic 9 (Node.js 24, Node-RED 5) ist als Vorabversion verfügbar - https://github.com/rdmtc/RedMatic/releases

---
*Written by Claude Fable on behalf of hobbyquaker.*
